custom_components/spook/ectoplasms/scene/repairs/unknown_entity_references.py

```diff
@@ -37,7 +37,7 @@ async def async_inspect(self) -> None:
         ].entities.values()
 
         for entity in scenes:
-            LOGGER.error("Scene: %s", entity.entity_id)
+            LOGGER.debug("Inspecting scene: %s", entity.entity_id)
```

Comment:
That is actually a leftover statement during testing.

This should not be in the code at all.


```suggestion
```